commcare_connect/templates/opportunity/opportunity_list.html

```diff
@@ -3,6 +3,63 @@
 
 {% block title %}{{ request.org }} - Opportunities{% endblock %}
 
+{% block inline_javascript %}
+  {{ block.super }}
+  <script>
+    function sortableTable() {
```

Comment:
We could of course do this using jQuery datatables but that will add few JS dependencies. So this is done without any dependencies using AlpineJS.